platform/kube/inject/inject_test.go

```diff
@@ -161,8 +165,9 @@ func TestIntoResourceFile(t *testing.T) {
 		}
 
 		config := &Config{
-			Policy:     InjectionPolicyEnabled,
-			Namespaces: []string{v1.NamespaceAll},
+			Policy:            InjectionPolicyEnabled,
+			IncludeNamespaces: []string{v1.NamespaceAll},
```

Comment:
This combination is disallowed by code you added [here](https://github.com/istio/pilot/pull/1437/files#diff-3f71ca8a1c3836bc4a3360d83741f2fbR196).

Reply:
Done